content/lore/elm-core-development.md

```diff
@@ -0,0 +1,186 @@
+---
+type: page
+title: "Lore: Elm Core Development"
+description: "Lore: Elm Core Development community history & cultural knowledge"
+published: true
+status: seedling
+authors: "@supermario,@gampleman"
+editors: "@jfmengels,@wolfadex"
+---
+
+<lore/>
+
+<tldr>TL;DR: Elm core dev (Evan) works differently, with tradeoffs some people dislike. Those conversations repeat. Core is considered stable, there are [no critical bugs or blockers](https://github.com/elm-janitor/manifesto). Rare [critical issues get attention](https://github.com/elm/virtual-dom/issues/168#issuecomment-1125076213). [Elm core design won't change for a while](https://discourse.elm-lang.org/t/where-can-we-find-the-roadmap-of-elm/6038/2). The [Elm community is very active](/).</tldr>
+
+
+
+> This article describes some of the peculiarities of Elm’s development model and their consequences. This model isn’t to everyone’s taste. We have noticed that there is often a desire to vocally criticise this model. That’s okay - there are certainly aspects worth criticising. We summarise or generalise some of these criticisms and try to respond to them, clearly delineating between productive and unproductive ways to move forward.
+>
+> After all, we've had these conversations before. This repeated dialogue causes a lot of stress and emotional work for everyone involved. This content has been written in the hope we can help future reincarnations of this dialogue resolve or advance, and help everyone find their place in it quickly and without unnecessary drama.
+
+---
+
+*Authored by non-core Elm community members: Mario Rogic and Jakub Hampl. Both Mario and Jakub have been in the Elm Community since 2016. Mario maintains Lamdera and Elmcraft. Jakub works on elm-visualization. This page expresses our private opinions; however we have some indication that these ideas are not uncommon among the more seasoned members of the community. Editors: Jeroen Engels, Wolfgang Schuster.*
```

Comment:
```suggestion
*Authored by non-core Elm community members: Mario Rogic and Jakub Hampl. Both Mario and Jakub have been in the Elm Community since 2016. Mario maintains [Lamdera](https://lamdera.com) and Elmcraft. Jakub works on [elm-visualization](https://package.elm-lang.org/packages/gampleman/elm-visualization/latest/). This page expresses our private opinions; however we have some indication that these ideas are not uncommon among the more seasoned members of the community. Editors: Jeroen Engels, Wolfgang Schuster.*
```